### ISSUE TITLE
Is there a way to distinguish players?

### ISSUE THREAD

**Issue by km19809** [kettle11/tangle#15]
I am impressed by this work.
It is working magically. (I've only tried the demo 
 and examples, though)
However, I could not find how to distinguish players.
For example, I need this feature to prohibit other players to control a player's character.
I have considered doing it on the JS side, but I could not get my identification from Tangle.

**Comment by kettle11** [comment]
Yes, there is a way but it's rather unintuitive and I'd like to improve it. 

This is the JavaScript code for the website's demo: https://github.com/kettle11/tangle_website/blob/main/index.ts

The `UserId` value is imported from the tangle library, and whenever that is passed in as an argument to a Wasm export export `Tangle` will automatically replace it with a unique number that represents the user.

**Comment by km19809** [comment]
Thank you, I'll try it.
I hope `UserId` is documented or included in a new example.
